# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-lt-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.16; lt-2.4.6)
libtoolize -c -f; aclocal; autoheader; autoconf; automake -a -c
./configure --disable-static --prefix=

# build
make

# install
install -d $DESTDIR/include/ogg
install -m 644 include/ogg/config_types.h $DESTDIR/include/ogg
install -m 644 include/ogg/ogg.h $DESTDIR/include/ogg
install -m 644 include/ogg/os_types.h $DESTDIR/include/ogg
install -d $DESTDIR/lib
install src/.libs/libogg.so.0.* $DESTDIR/lib/libogg.so.0
ln -sf libogg.so.0 $DESTDIR/lib/libogg.so
install -d $DESTDIR/lib/pkgconfig
install -m 644 ogg.pc $DESTDIR/lib/pkgconfig
install -d $DESTDIR/share/aclocal
install -m 644 ogg.m4 $DESTDIR/share/aclocal

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/libogg-build.pc << EOF
Name: libogg
Version: 1.3.5
Description: Ogg media container library
Requires: $REQUIRES

devel=\\
/include/ogg/config_types.h \\
/include/ogg/ogg.h \\
/include/ogg/os_types.h \\
/lib/libogg.so \\
/lib/pkgconfig/ogg.pc \\
/share/aclocal/ogg.m4

exec=\\
/lib/libogg.so.0
EOF

exit

# patch
--- libogg-1.3.5.orig/configure.ac
+++ libogg-1.3.5/configure.ac
@@ -7,7 +7,7 @@
 AC_CONFIG_SRCDIR(src/framing.c)
 
 AM_INIT_AUTOMAKE
-AM_MAINTAINER_MODE([enable])
+AM_MAINTAINER_MODE
 
 dnl Library versioning
 
# source
http://downloads.xiph.org/releases/ogg/libogg-1.3.5.tar.gz
